cps/main.c

```diff
@@ -688,10 +688,6 @@ kni_create(struct rte_kni **kni, struct rte_mempool *mp,
 	ops.port_id = conf.group_id;
 	ops.change_mtu = kni_change_mtu;
 	ops.config_network_if = kni_change_if;
-	/*
-	 * XXX This must be implemented to change KNI MAC address.
-	 * ops.config_mac_address = kni_config_mac_address;
```

Comment:
Doesn't CPS assume that the MAC address of the KNI interface is the same of the real NIC? If so, we must implement this function to forbid the change.

Reply:
We could forbid this change only if we want to assume that we never need to change the MAC address of the real NIC. I'm OK with this assumption but I don't necessarily see the motivation behind it.

Leaving in the default configuration function just means that we can, if needed, make the KNI also change its MAC address if the real NIC's MAC address needs to be changed.